contracts/scripts/native_solc_compile_all_logpoller

```diff
@@ -29,4 +29,5 @@ compileContract () {
 }
 
 
-compileContract tests/LogEmitter.sol
\ No newline at end of file
+compileContract tests/LogEmitter.sol
+compileContract tests/VRFLogEmitter.sol
```

Comment:
Should probably only be in the vrf compile?

Reply:
This is just testing the LogPoller specific functionality and is not specific to VRF. I named it VRFLogEmitter, because the VRF listener listens to specific VRF events, which I had to mimic in this test contract to check if unfulfilled requests in blocks older than finalityDepth are correctly handled..